src/encoded/static/components/genome_browser.js

```diff
@@ -497,11 +507,13 @@ class GenomeBrowser extends React.Component {
         let files = propsFiles;
 
         // Apply sort parameters
-        orderedSortParam.forEach((param) => {
-            files = _.chain(files)
-                .sortBy(obj => sortLookUp(obj, param));
-        });
-        files = files.value();
+        if (this.props.displaySort) {
+            orderedSortParam.forEach((param) => {
+                files = _.chain(files)
+                    .sortBy(obj => sortLookUp(obj, param));
+            });
+            files = files.value();
+        }
```

Comment:
I think this is ok. I added in displaySort so that we could turn off the sort buttons and leave the cart sorting. Currently the carts do sort results, just on a page-by-page basis and we just don't bring the users' attention to it. This all needs to be revisited when sorting buttons are added to the cart so it seems ok to get rid of page-by-page sorting for now. The comment explanation of displaySort is now misleading, though, because it's not just turning off the sort buttons, it's now turning off the sort entirely.

![Screen Shot 2020-10-08 at 8 58 10 AM](https://user-images.githubusercontent.com/2887343/95484358-3b774e00-0945-11eb-9a84-1f19e9129084.png)

Reply:
OK, I see.